tests/Integration/Framework/Test_AuthDB.py

```diff
@@ -0,0 +1,200 @@
+""" This is a test of the AuthDB
+    It supposes that the DB is present and installed in DIRAC
```

Comment:
This test is not added to all_integration_server_tests.sh

Reply:
added now